src/ansys/fluent/core/services/field_data.py

```diff
@@ -354,6 +356,19 @@ def __call__(self, *args, **kwargs):
         return self._field_data_accessor(*args, **kwargs)
 
 
+def _surface_converter(old_args_dict, new_args):
+    surfaces_list = []
+    if "surface_name" in old_args_dict:
+        surfaces_list.append(old_args_dict["surface_name"])
```

Comment:
`surface_name` is mentioned here but not in the old interface unless this function is being used in different contexts (but don't do that).

Reply:
This is actually being used by both field_data and field_transaction. That's why a common method.